operator/pkg/model/translation/gateway-api/translator.go

```diff
@@ -25,12 +25,15 @@ const (
 
 type translator struct {
 	secretsNamespace string
+
+	idleTimeoutSeconds int
 }
 
 // NewTranslator returns a new translator for Gateway API.
-func NewTranslator(secretsNamespace string) translation.Translator {
+func NewTranslator(secretsNamespace string, idleTimeoutSeconds int) translation.Translator {
```

Comment:
It's fine for this one, but the next time we need to add another parameter here, we should consider moving to something more extensible, so we don't need to change the function signature. Maybe a variadic options struct or something?